Makefile

```diff
@@ -2,23 +2,23 @@ PREFIX         ?= /usr
 
 #CXX            ?= clang++
 CXX            ?= g++
-FFLAGS         ?=
+RM             ?= rm -f
```

Comment:
Make already defines "RM = rm -f" implicitly, no need to overwrite it with the same thing. 
(to see all implicit defs run `make -p`)

Though, being explicit doesn't hurt.
Thoughts?

Reply:
I personally dislike implicit rules in general.  Especially the GNU Make implicit compilation rules.  Implicit variables perhaps are fine.  Is there any reason not to just be explicit here?

Reply:
Not really.